### PR TITLE
try noble chiseled sdk

### PR DESCRIPTION
**Pull request by sahma19**

### CODE REVIEW

File: domains/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
-# Run the certificates project with --swagger
 RUN dotnet run --project domains/certificates/Query.API/API/API.csproj --swagger
-# Run the proxy project with --swagger
+
 RUN dotnet run --project domains/authorization/Proxy/Proxy.csproj --swagger
+
+FROM mcr.microsoft.com/dotnet/runtime-deps:8.0-noble-chiseled AS final
+
+WORKDIR /swaggerfiles
```
Comment (ckr123) [inline domains/Dockerfile:14]: Why this change? You will have to change it in EO base as well

File: domains/Dockerfile [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+
+COPY --from=build /app/domains/certificates/Query.API/API/contracts.yaml ./contracts.yaml
```
Comment (ckr123) [inline domains/Dockerfile:16]: What does from=build do?

File: domains/Dockerfile (hunk 1 above)
Comment (sahma19) [reply domains/Dockerfile]: Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1381/commits/d64112003eb8f3ea93f7a1df279f779d11ca677b

File: domains/Dockerfile (hunk 2 above)
Comment (sahma19) [reply domains/Dockerfile]: Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1381/commits/d64112003eb8f3ea93f7a1df279f779d11ca677b

**Review by ckr123 — APPROVED** [review]
